src/common/components/clusterConfiguration/DiskEncryptionFields/DiskEncryptionControlGroup.tsx

```diff
@@ -45,24 +45,30 @@ const DiskEncryptionControlGroup: React.FC<DiskEncryptionControlGroupProps> = ({
     }
   }, [isSNO, setFieldValue]);
 
+  const disableMessage = 'This option is not editable after the draft cluster is created';
+
   return (
     <Stack hasGutter>
       <StackItem>
-        <SwitchField
-          name="enableDiskEncryptionOnMasters"
-          label={`Enable encryption of installation disks${
-            !isSNO ? ' on control plane nodes' : ''
-          }`}
-          isDisabled={isDisabled}
-        />
-      </StackItem>
-      <RenderIf condition={!isSNO}>
-        <StackItem>
+        <Tooltip hidden={!isDisabled} content={disableMessage}>
```

Comment:
Please use the SwitchField's `tooltipProps` prop in order to add the tooltip
Ref: https://github.com/openshift-assisted/assisted-ui-lib/blob/92fc64999f845b4e6920d939428d0c49c2e039c8/src/common/components/ui/formik/SwitchField.tsx#L17